components/discord_bot/actions/add-role/add-role.mjs

```diff
@@ -0,0 +1,57 @@
+import common from "../common.mjs";
+
+const { discord } = common.props;
+
+export default {
+  ...common,
+  key: "discord_bot-add-role",
+  name: "Add Role",
+  description: "Assign a role to a user. Remember that your bot requires the `MANAGE_ROLES` permission. [See the docs here](https://discord.com/developers/docs/resources/guild#add-guild-member-role)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    discord,
+    guildId: {
+      ...common.props.guildId,
+    },
+    userId: {
```

Comment:
Make sure to include all common props. Then you don't need to list discord or guildId here since they're already in common.props.
```
props: {
  ...common.props,
   userId: {
```